### PR TITLE
Fix some NLS issues

### PR DESCRIPTION
**Pull request by kgaillot**

### CODE REVIEW

**Comment by kgaillot** [comment]
@clumens , can you review this when you get a chance? Thanks

**Comment by kgaillot** [comment]
This doesn't seem sufficient to get it working yet, but the default-to-disabled and gitignore commits are important to get in soon

File: lib/common/logging.c [hunk 1]
```diff
@@ -822,6 +822,25 @@ crm_log_preinit(const char *entity, int argc, char **argv)
             set_format_string(lpc, crm_system_name, pid, nodename);
         }
     }
+
+#ifdef PCMK__SUPPORT_NLS
```
Comment (clumens) [inline lib/common/logging.c:826]: This should just be ENABLE_NLS.

**Comment by clumens** [comment]
I think in the future, you can use `make -C po update-po` to update the translations, and something like `make -C po pacemaker.pot` to update the pot file.

File: lib/common/logging.c (hunk 1 above)
Comment (kgaillot) [reply lib/common/logging.c:826]: Aha, that's probably the issue

Comment (kgaillot) [reply lib/common/logging.c]: rebased on current main, only change is this fix

**Comment by clumens** [comment]
Looks good.

**Pull request merged**